lib/lib.wrappers.php

```diff
@@ -14,6 +14,15 @@ function dj_json_decode(string $str)
     return json_decode($str, true, 512, JSON_THROW_ON_ERROR);
 }
 
+/**
+ * Try decode a JSON string with our preferred settings.
+ * Does not throw error, but errors can be obtained via json_last_error().
+ */
+function dj_json_try_decode(string $str)
+{
+    return json_decode($str, true, 512);
```

Comment:
The `512` argument was only there to be able to specify the next, so might be better left out. Or maybe make `JSON_THROW_ON_ERROR` dependent on an optional parameter to `dj_json_decode`. @meisterT , @thijskh WDYT?

Reply:
Removed `512`. IMO, `dj_json_try_decode` seems clearer than `dj_json_try_decode($str, 0)`.